modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/checkbox/CheckboxDDMFormFieldTemplateContextContributor.java

```diff
@@ -48,6 +48,11 @@ public Map<String, Object> getParameters(
 			ddmFormField.getProperty("localizedObjectField"));
 
 		return HashMapBuilder.<String, Object>put(
+			"editOnlyInDefaultLanguage",
+			ddmFormField.getProperty("editOnlyInDefaultLanguage")
+		).put(
+			"isLocalizable", ddmFormField.getProperty("isLocalizable")
+		).put(
```

Comment:
Both properties are boolean:
```
ddmFormField.getProperty("editOnlyInDefaultLanguage")
ddmFormField.getProperty("isLocalizable")

```
Can you put this `GetterUtil.getBoolean(ddmFormField.getProperty("abcdef"))` ?
Please, do this in all usages.

Reply:
uhmmm, I see! To make these parameters resilient to null and non-boolean values, right?

```
	/**
	 * Returns the Object value as a boolean. If the value is <code>null</code>,
	 * the {@link #DEFAULT_BOOLEAN} is returned. If the value does not match a
	 * {@link #BOOLEANS} value, <code>false</code> is returned.
	 *
	 * @param  value the value to convert
	 * @return the value as a boolean
	 */
	public static boolean getBoolean(Object value) {
		return getBoolean(value, DEFAULT_BOOLEAN);
	}
```